### PR TITLE
Update/16102018

### PR DESCRIPTION
**Pull request by rllin-fathom**
paired with https://github.com/medicode/diseaseTools/pull/3231

1. pass through new arg `use_tpu` in https://github.com/medicode/tensor2tensor/pull/105/commits/0d2293f7eb454077634c39b8cc76264245ab2417
1. deal with fathom logits with a nonce https://github.com/medicode/tensor2tensor/pull/105/commits/27ee8fcb6a4915e47723bbc7b8fd608eab31c341
1. skip new tests that upstream also skips

### CODE REVIEW

**Comment by cbockman** [comment]
force merging because depends on paired main repo pr (3231) and is failing in tests b/c of this
